libcloud/dns/drivers/nsone.py

```diff
@@ -206,7 +206,7 @@ def delete_record(self, record):
 
         return response.status == httplib.OK
 
-    def create_record(self, name, zone, type, data, extra=None):
+    def create_record(self, name, zone, type_, data, extra=None):
```

Comment:
``type`` argument is part of the standard API (aka is the same for all the provider drivers) so we should use it here as well.

I know it's not ideal since it clashes with a built-int value, but not much we can do at this point (besides changing the standard API + updating all the drivers in the future).

Reply:
Besides that, changes look good to me - please just add some tests when you get a chance.